patches/tModLoader/Terraria/release_extras/DedicatedServerUtils/manage-tModLoaderServer.sh

```diff
@@ -279,34 +253,33 @@ function print_help {
 
 Usage: script.sh COMMAND [OPTIONS]
 
+ENV Variables:
+ STEAMCMDPATH        Custom path for the steamcmd binary if your package manager does not have it
+ TMLVERSION          TML version to download. By default this is the latest release
+
 Options:
  -h|--help           Show command line help
  -v|--version        Display the current version of the management script
  -g|--github         Download tML from Github instead of using steamcmd
  -f|--folder         The folder containing all of your server data (Mods, Worlds, serverconfig.txt, etc..)
  -u|--username       The steam username to login use when downloading tML. Not required to download mods
- --tml-version       The version of tML to download from Github. Needs to be an exact tag name (eg. "v2022.06.96.4")
- --steamcmdpath      Custom steamcmd path for users who do not have a installation in PATH.
- --keepbackups       Will keep all tML backups instead of the most recent one when updating
+ --keepbackups       When installing with --github, keep all previous versions instead of deleting them when updating
+ --tmlversion        Version of tModLoader to install. Only works if --github is provided. Functionally equivalent to the TMLVERSION env variable
```

Comment:
Should probably keep the legacy arg as well, no need to break things unnecessarily